lib/removeFingerprint/index.js

```diff
@@ -1,17 +1,29 @@
-const {basename} = require('path');
+const { basename } = require('path');
 const finger = require('fingerprinting');
-const {readFile} = require('../afsync');
+const { readFile } = require('../afsync');
+
+/**
+ * Removes the hashed part of a file path string
+ * @param  {String} path File path
+ * @return {String}
+ */
+const removeHash = path => (
+    path.replace(/\.[a-f0-9]{20}\.(js|css|js.LICENSE.txt)$/, '.$1')
+);
 
 /**
  * Removes the fingerprint part of a file path string
  * @param  {String} path File path
  * @return {String}
  */
 module.exports = async function removeFingerprint(path) {
-	const content = await readFile(path);
-	const {id} = finger(
-		basename(path),
-		{content}
-	);
-	return path.split('.').filter(part => !id.startsWith(part)).join('.');
+    const content = await readFile(path);
+    const {id} = finger(
+        basename(path),
+        {content}
+    );
+
+    const cleanedPath = path.split('.').filter(part => !id.startsWith(part)).join('.');
```

Comment:
This module already searches for the fingerprint in the filename, since this is another pattern of a fingerprint, Do you think we use the existing mechanism for removal? Or do you think it presents a risk to a filename that would have 20 characters of hexadecimal values?
```js
const FINGERPRINT_PATTERN = /[a-f0-9]{20}/;

return path.split('.')
	.filter(part => !id.startsWith(part))
	.filter(part => !FINGERPRINT_PATTERN.test(part)) // <= new pattern
	.join('.')
```

Reply:
I thought about this too when I was doing this PR and my assumption was that it could be an issue - but now that I'm looking at the files, I'm kinda changing my mind.  

We can hit 20 chars easily, but not without a special character which would fail the regex - eg. `apps/MadeOnFiverrrrr.modern.a02bfed9340f88ac11a5.js` has `apps/MadeOnFiverrrrr` and `a02bfed9340f88ac11a5` which are 20 chars, but only the latter gets stripped.

I'm fine either way.

Reply:
Yeah, that's what I was thinking. Who names their file with 20 consecutive lowercase a-f and numbers? And if they do, I'm willing to bet they're gonna have other issues from more important tools.